website/docs/guides/features-block.html.markdown

```diff
@@ -213,6 +213,6 @@ The `virtual_machine_scale_set` block supports the following:
 
 ~> **Note:** Support for Force Delete is in an opt-in Preview.
 
-* `roll_instances_when_required` - (Optional) Should the `azurerm_linux_virtual_machine_scale_set` and `azurerm_windows_virtual_machine_scale_set` resources automatically roll the instances in the Scale Set when Required (for example when updating the Sku/Image). Defaults to `true`.
+* `roll_instances_when_required` - (Required) Should the `azurerm_linux_virtual_machine_scale_set` and `azurerm_windows_virtual_machine_scale_set` resources automatically roll the instances in the Scale Set when Required (for example when updating the Sku/Image).
```

Comment:
Can we include the default value here to be consistent?
```suggestion
* `roll_instances_when_required` - (Required) Should the `azurerm_linux_virtual_machine_scale_set` and `azurerm_windows_virtual_machine_scale_set` resources automatically roll the instances in the Scale Set when Required (for example when updating the Sku/Image). Defaults to `false`. 
```
(since bools are `false` if not explicitly set to `true`)

Reply:
I've taken another thought when seeing this default. Usually, a `Required` property is not supposed to have a default value. And the default value of this is already set in `features/defaults.go`:
https://github.com/hashicorp/terraform-provider-azurerm/blob/fe5fee88b6beece47f79e09719535b917b163f53/internal/features/defaults.go#L53
So I think we should still fix the schema instead. By changing a `Required` prop to `Optional`, user won't be seeing a breaking change. Earlier, when user want to use this feature flag, they will explicitly set it, which is also fine with this schema update. If user doesn't explicitly set this feature flag, then everything remains same with this new change, the feature is still enabled.